index.html

```diff
@@ -375,8 +375,7 @@
 		
 		<footer>
 			<div>
-				<p>We love feedback! If you have any questions, please send us an email <a href='mailto:albuquerque@codeforamerica.org'>albuquerque@codeforamerica.org</a><br>
-          <span class='tiny'>Built by Miami-Dade team. Forked by the <a href="http://www.codeforamerica.org/governments/albuquerque/">CfA ABQ</a> team with love and hugs and stuff.</p>
+				<p>The Code for America Albuquerque team is piloting this prototype with your company to help get you the tools you need to be happy financially. If you have any questions, please send us an email <a href='mailto:albuquerque@codeforamerica.org'>albuquerque@codeforamerica.org</a>.</p>
```

Comment:
@boonrs I spent a while last night working on styling the radios and had to dump it so all this does it make them inline. I think that's pretty straight forward. I need to merge this to get the pilot out and I am going to do that now.